tokio/src/runtime/dump.rs

```diff
@@ -0,0 +1,69 @@
+//! Snapshots of runtime state.
+
+use std::fmt;
+
+/// A snapshot of a runtime's state.
+#[derive(Debug)]
+
+pub struct Dump {
+    tasks: Tasks,
+}
```

Comment:
There appears to be a formatting issue here, but for some reason it is not caught by rustfmt?